x/feemarket/types/keys.go

```diff
@@ -14,7 +14,8 @@ const (
 
 // prefix bytes for the feemarket persistent store
 const (
-	prefixBlockGasUsed = iota + 1
+	prefixBlockGasUsed      = iota + 1
+	deprecatedPrefixBaseFee // nolint
```

Comment:
how does this work?

Reply:
iota? it's an enum that starts from 0

Reply:
I mean `deprecatedPrefixBaseFee` - we need to keep these bytes around for this deprecated parameter? why is it a deprecated param being introduced here?

Reply:
it's the old byte used for the BaseFee prefix. I added it to prevent someone from adding the same prefix as the key for another value